client/script.js

```diff
@@ -1,83 +0,0 @@
-const searchBtn = document.getElementById('google-search')
-
-searchBtn.addEventListener('click', search);
-
-function search(e) {
-    try {
-    e.preventDefault();
-
-    //Reset Links Section for next search 
-    let updatedLinkSection = document.getElementById('link-section')
-    updatedLinkSection.innerHTML = "";
-
-    const searchedTerm = document.getElementById('search').value;
-    let searchArray = searchedTerm.split(' ');
-    console.log(searchedTerm)
-    
-    console.log(searchArray)
-    fetch('http://localhost:3000/search')
-    .then(resp => resp.json())
-    .then(data => {
-        for (let k = 0; k < searchArray.length; k++) {
-            sortResults(data, searchArray[k])
-        } 
-        return data
-    })
-    .then(orderArray)
-    
-    } catch (error) {
-        console.log(error);
-    }
-  }
-
-  function sortResults(data,word) {
-    let resData = data;
-    console.log(word)
-    if (word[0]) {
-
-    for (let i = 0; i <= 9; i++) {
-
-        let str = resData[i]["description"].toLowerCase();
-        let str2 = resData[i]["title"].toLowerCase();
-
-        if (str.includes(word.toLowerCase())) {
-            console.log('butter'); 
```

Comment:
lmao

Reply:
is this working as intended? e.g., if I search 'netflixd', should it return netflix as the top result?